### ISSUE TITLE
Inappropriate => Indentation Warning in Hash

### ISSUE THREAD

**Issue by pioto** [rodjek/puppet-lint#698]
Here's the sample manifest:

```puppet
class test {
    test { 'foo':
        bind => {
            "${test}:443" => ['test'],
            "${test}:80"  => [],
        },
    }
}
```

The warnings which (IMHO) are erroneous:

```
WARNING: indentation of => is not properly aligned (expected in column 27, but found it in column 23) on line 4
WARNING: indentation of => is not properly aligned (expected in column 27, but found it in column 24) on line 5
```

It seems like it's being thrown off by the colons or... something in the hash keys.

**Comment by dbeckham** [comment]
This is probably the same issue I encountered in #697.  @pioto can you post your Gemfile and Gemfile.lock, as well as what Ruby version (and patchlevel) you are running when this happens?

**Comment by pioto** [comment]
Here's my Gemfile & Gemfile.lock. I saw this with:

```
$ ruby --version
ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]
```

[puppet-lint-gemfile.zip](https://github.com/rodjek/puppet-lint/files/953026/puppet-lint-gemfile.zip)

**Comment by rodjek** [comment]
Fixed in #701
